### ISSUE TITLE
Blips and objects are not destroyed on resource stop clientside

### ISSUE THREAD

**Issue by xxshady** [altmp/altv-issues#1633]
### Description of the problem

If you create blip or object clientside it will not be destroyed when resource stops. WebView is destroyed, same as Checkpoint.

### Reproduction steps

```js
const blip = new alt.PointBlip(0, 0, 0) // not destroyed
const obj = new alt.Object("sm_prop_smug_crate_s_jewellery", alt.Player.local.pos, alt.Vector3.zero) // not destroyed
const webview = new alt.WebView("google.com") // will be destroyed
```

### Expected behaviour

All baseobjects should be destroyed on resource stop to keep the consistency of the API.

### Operating system

Windows 11

### Version

13.0-dev12

**Comment by litolax** [comment]
Confirmed on 13.0-dev13

**Comment by LeonMrBonnie** [comment]
Needs a change in SDK because currently blips and objects are created without an associated resource

**Comment by C0kkie** [comment]
Will be fixed in dev/14.0-dev55
